cluster-autoscaler/cloudprovider/oci/nodepools/consts/annotations.go

```diff
@@ -25,4 +25,7 @@ const (
 	// ToBeDeletedByClusterAutoscaler is the taint used to ensure that after a node has been called to be deleted
 	// no more pods will schedule onto it
 	ToBeDeletedByClusterAutoscaler = "ignore-taint.cluster-autoscaler.kubernetes.io/oke-impending-node-termination"
+
+	// EphemeralStorageSize is the freeform tag key that would be used to determine the ephemeral-storage size of the node
+	EphemeralStorageSize = "cluster-autoscaler/node-ephemeral-storage"
```

Comment:
Why are we using free-form tags to provide local disk information? Is there something special about the scale from `0`  scenario that prevents the OKE and InstancePool implementations from using the `LocalDisksTotalSizeInGBs` and `LocalDisks` fields of  [core.shape](https://github.com/oracle/oci-go-sdk/blob/master/core/shape.go#L27) when populating the node template in `buildNodeFromTemplate`?

Reply:
Assuming ephemeral storage, like other resources, can be determined from the shape, it still useful to optionally support using free-form tags to _override_ shape specific values like cpu, memory, ephemeral-storage, gpu, etc..

However, IMO, the values from [core.shape](https://github.com/oracle/oci-go-sdk/blob/master/core/shape.go#L27) should remain the _default_ source resource information for the node when available.

Reply:
OK... after looking into this more, it looks like [most shapes](https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm) use "block storage only". This means that _most of the time_ `LocalDisksTotalSizeInGBs` and `LocalDisks` will not be set. 

I still think it's a good idea for both the node-pool and instance-pool implementations to check for and use `shape.LocalDisksTotalSizeInGBs` from [core.shape](https://github.com/oracle/oci-go-sdk/blob/master/core/shape.go#L27) by default. However, given that it's going to be unset most of the time, it seems like a good idea to add the ability to receive these values via free-form tags.